docs/release_notes/v0.8.0.md

```diff
@@ -0,0 +1,294 @@
+
+**Added** docs authoring guideline
+
+**Added** pubsub custom routes
+
+**Added** liveness config annotation
+
+**Updated** actor spec to remove activation api
+
+**Added** how to enable tls for redis cache (Thanks @robinmanuelthiel)
+
+**Added** token-based authentication
+
+**Added** more exampels to Azure Event Grid binding (Thanks @Masahigo)
+
+**Added** bi-directional bindings howto
+
+**Added** mqtt pubsub howto (Thanks @kumaya)
+
+**Updated** dapr run cli to use cli components path
+
+**Added** apache pulsar pubsub howto (Thanks @elyst)
+
+**Added** snssqs pubsub howto (Thanks @khous)
+
+**Updated** missing steps for eventhubs
+
+**Added** use-cases to concept docs (Thanks @arschles)
+
+**Added** W3C trace context howtos
+
+**Added** Grafana Actor Dashboard
+
+**Added** how to use multi fragment path for service invocation
+
+
+## Tests
+
+**Added** alerts for control plane and daprd for cpu/mem
+
+**Refactor** longhaul tests apps to use shared library
+
+
+## Upgrading to Dapr 0.8.0
+
+If you're upgrading from an older version of Dapr to 0.8.0, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have:
+
+```bash
+dapr uninstall --all
+```
+
+Next, get the latest CLI following [these](https://github.com/dapr/cli#installing-dapr-cli) instructions, or alternatively download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have downloaded the CLI, run:
+
+```bash
+dapr init
+```
+
+Wait for the update to finish, and you're good to go!
+Make sure you have the right runtime version (0.8.0) with:
+
+```bash
+dapr --version
+```
+
+### Kubernetes
+
+Download the latest CLI release as outlined above in the `Local Machine / Self-hosted` section.
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+```bash
+dapr uninstall --kubernetes
+```
+
+It's fine to ignore any errors that might show up.
+
+If you previously installed Dapr using __Helm 2.X__:
+
+```bash
+helm del --purge dapr
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+```bash
+helm repo update
+```
+
+Wait until Dapr is uninstalled, and make sure the `dapr-system` namespace is gone.
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, if you want to install Dapr in a dev/test setup, run:
+
+```
+dapr init --kubernetes
+```
+
+#### Post installation
+
+After Dapr 0.8.0 has been installed, perform a rolling restart for your deployments.
+
+## Breaking Changes
```

Comment:
Need to add the change in the components folder automatic creation flow here. It can be debated if that is exactly breaking but is a change to user assumptions so I think would not hurt to recommunicate here